index.dd

```diff
@@ -240,6 +240,34 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE_RUNNABLE Generate a config parser at Compile-Time,
+----
+import std.algorithm, std.conv, std.range, std.stdio, std.traits;
+struct Config {
+    int a, b;
+    string c;
+}
+void main() {
+    Config conf;
+    // use the generated parser for each entry
+    "b=1,a=2,c=hello"
+        .splitter(",")
+        .each!(e => conf.read(e));
```

Comment:
`"b=1,a=2,c=hello".splitter(",").each!(e => conf.read(e));`

That would get rid of 2 lines. The syntax coloring will help to make it readable.